samcli/lib/bootstrap/bootstrap.py

```diff
@@ -35,6 +36,28 @@ def manage_stack(profile, region):
     return bucket_name
 
 
+def print_managed_s3_bucket_info(
+    s3_bucket: str, bold: Optional[bool] = False, show_s3_bucket_option: Optional[bool] = True
+):
+    """
+    Print information about the managed S3 bucket.
+
+    Parameters
+    ----------
+    s3_bucket : str
+        The name of the managed S3 bucket
+    bold : bool, optional
+        Whether to print the message in bold, by default False
+    show_s3_bucket_option : bool, optional
+        Whether to show the s3-bucket option message, by default True
+    """
+    message = f"\n\tManaged S3 bucket: {s3_bucket}"
+    click.secho(message, bold=bold)
+    click.echo("\tA different default S3 bucket can be set in samconfig.toml")
```

Comment:
I thinking this is an opportunity to improve on the message as well. 
```suggestion
    click.echo("\tTo use a specify bucket, specify `resolve_s3 = false` and `s3_bucket= <bucket arn>`")
```